src/bp/core/api.ts

```diff
@@ -312,7 +313,10 @@ export class BotpressAPIProvider {
       security: this.security,
       experimental: this.experimental,
       workspaces: this.workspaces,
-      distributed: this.distributed
+      distributed: this.distributed,
+      NLU: {
```

Comment:
don't you have to define type just like we do from line 227 to 243 ?

Reply:
yeah, theses lines are for properties wich are then returned when calling `create`. For the engine, as I didnt want to change current behavior of the code, Instead of exposing an already instanciated property, I expose the whole class and let the nlu-wrapper(module) call the egine ctor himself.

This could sure be changed in the future but it implies quite a refactor. I personnaly wouldnt change it in this PR.

Reply:
can you add a `//TODO: NLU` so we don't forget about this?